index.html

```diff
@@ -0,0 +1,172 @@
+                  <li><a class="menuLink" href="#">Program</a></li>
+                  <li><a class="menuLink" href="about.html">About</a></li>
+                  <li><a class="menuLink" href="#">Join</a></li>
+                </ul>
+                
+                <ul class="topList">
+                    <li><i class="fa fa-facebook-f"></i></li>
+                    <li><i class="fa fa-twitter"></i></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+
+        </nav>
+        <nav class="desktopNavFirst">
+            
+
+           <ul class="secondNavList">
+               <li class="secondNavItem"><a href="./about.html">About</a></li>
+               <li class="secondNavItem">Program</li>
+               <li class="secondNavItem">Join</li>
+               <li class="secondNavItem">Sponsor</li>
+               <li class="secondNavItem">News</li>
+               <li><button class="summitBtn" type="button">Summit</button></li>
+           </ul>
+          <a href="./index.html"> <img src="./images/images.png" alt="test"> </a>  
+        </nav>
+    </header>
+    <section class="intro-Section">
+        <hgroup>
+            <h1 class="firstHeading">"Hello! Sharing World"</h1>
+            <h1 class="secondHeading">Women in the World Summit 2021</h1>
+    
+        </hgroup>
+        <p>
+            A joufyl celebration believing in the value of opennes and
+            sharing, creating a positive change with people from over 
+            80 countries is taking place in October, in America.
+        </p>
+        <div class="placeTime">
+            <h1>2021.10.11(MON) ~ 15(FRI)</h1>
+            <p>@ National Museum of Korea, Art Center Nabi and more</p>
+        </div>
+    </section>
+    <section class="program">
+        <h1>Main Program</h1>
+        <hr>
+        <div class="programDetail">
+            <ul class="firstDetail">
+                
+                <li><img src="./images/Group.png" alt="Lecture-logo"></li>
+                <li><h1>Lecture</h1></li>
+                <li><p> Listen to the speakers from various countries about the messeges of sharing and opening</p></li>
+
+            </ul>
+            <ul class="firstDetail">
+                
+                <li><img src="./images/group2.svg" alt="Exhibition"></li>
+                <li><h1>Exhibiton</h1></li>
+                <li><p> Passion, leadership, and action are the three themes of this event, which will ultimately lead you to hero status</p></li>
+
+            </ul>
+            <ul class="firstDetail">
+                
+                <li><img src="./images/conversation.svg" alt="Conversation"></li>
+                <li class="headingList"><h1>Forum</h1></li>
+                <li><p> Have the time to share your thoughts
+                    and opinions with experts for
+                    each topic</p></li>
+            </ul>
```

Comment:
- [ ] Kindly adjust this section on mobile screens to be properly aligned. You also have a horizontal overflow on this screen size.

_See images below_

**Original** | **Your project**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/125055295-e2675280-e05b-11eb-95b3-b25670ee6116.png) | ![image](https://user-images.githubusercontent.com/49621279/125055393-fe6af400-e05b-11eb-953a-f88d8092ccac.png)